### PR TITLE
MM-31032 Docs on rolling out new HA server versions to prod

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://mattermost.atlassian.net/browse/MM-31032

Updated:
- Admin Guide > Scaling Mattermost > High Availability Cluster (E20) > Troubleshooting: added new subsection "Capturing High Availability Troubleshooting Data"

### CODE REVIEW

File: source/deployment/cluster.rst [hunk 1]
```diff
@@ -431,6 +431,19 @@ For example, if you're deploying Mattermost on AWS with Amazon Aurora we recomme
 Troubleshooting
 ---------------
 
+Capturing High Availability Troubleshooting Data
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When deploying Mattermost in a High Availability configuration, we recommend that you keep Prometheus and Grafana metrics as well as cluster server logs for as long as possible and for at least two weeks.
```
Comment (justinegeffen) [inline source/deployment/cluster.rst:437]:
```suggestion
When deploying Mattermost in a High Availability configuration, we recommend that you keep Prometheus and Grafana metrics as well as cluster server logs for as long as possible - and at minimum two weeks. 
```

**Comment by amyblais** [comment]
Is this specific to the v5.30 release? If not, any docs unrelated to a release can be submitted to the ``master`` branch.

**Review by justinegeffen — APPROVED** [review]

**Comment by amyblais** [comment]
@cwarnermm ^

**Comment by cwarnermm** [comment]
Thanks, @amyblais, for that clarification! I've confirmed with @wiersgallak that this particular update isn't directly tied to the v5.30 release, but that there is value in this docs addition being available sooner than later.

File: source/deployment/cluster.rst [hunk 2]
```diff
@@ -431,6 +431,19 @@ For example, if you're deploying Mattermost on AWS with Amazon Aurora we recomme
...
+Capturing High Availability Troubleshooting Data
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When deploying Mattermost in a High Availability configuration, we recommend that you keep Prometheus and Grafana metrics as well as cluster server logs for as long as possible - and at minimum two weeks. 
+
+You may be asked to provide this data to Mattermost for analysis and troubleshooting purposes.
+
+.. note::
+
+  - Ensure that server log files are being created. See :ref:`troubleshooting` for details.
```
Comment (wiersgallak) [inline source/deployment/cluster.rst:443]: @cwarnermm Wondering what the ```:ref: `troubleshooting` ``` refers to here?

Comment (cwarnermm) [reply source/deployment/cluster.rst:443]: @wiersgallak The intention was to provide a link to https://docs.mattermost.com/install/troubleshooting.html#review-mattermost-logs; however, I suspect that my relative link may not work as intended. @justinegeffen for this internal link, is the ```#review-mattermost-logs``` also required to resolve correctly?

Comment (cwarnermm) [reply source/deployment/cluster.rst]: @wiersgallak & @justinegeffen - I've fixed this relative link by replacing it with a hypertext link for now.

**Comment by amyblais** [comment]
Preparing to merge v5.30 docs into master - can open a new PR for any further edits.

**Pull request merged**